xcom2-launcher/xcom2-launcher/Forms/SettingsDialog.cs

```diff
@@ -129,6 +127,8 @@ private void bOK_Click(object sender, EventArgs e)
             Settings.CheckForPreReleaseUpdates = checkForPreReleaseUpdates.Checked;
             GlobalSettings.Instance.IsSentryEnabled = useSentry.Checked;
             Settings.AllowMultipleInstances = allowMutipleInstances.Checked;
+            Settings.GamePath = gamePathTextBox.Text;
+            Settings.ModPaths = modPathsListbox.Items.Cast<string>().ToList();
```

Comment:
You should probably save `modPathsListbox.Items.Cast<string>().ToList()` to a variable since you do it twice in the same function. Otherwise looks good